conf/config-default.yaml

```diff
@@ -128,6 +128,7 @@ nginx_config:                     # config for render the template to genarate n
   #envs:                            # allow to get a list of environment variables
   #  - TEST_ENV
   http:
+    access_log_switch: on          # on or off, default on
```

Comment:
Better to use `enable_access_log: true` to ensure consistency.

Reply:
yeah, right